### ISSUE TITLE
Bump bitarray dependency

### ISSUE THREAD

**Issue by lcsvcn** [ethereum/eth-account#148]
Latest bitarray is 2.4.0. eth-account is still using < 1.3.0

It is good to keep lib update, for security and stability reasons.

```
The conflict is caused by:
    The user requested bitarray==2.4.0
    eth-account 0.6.1 depends on bitarray<1.3.0 and >=1.2.1

```

**Comment by sudhamjayanthi** [comment]
+1

due to the old version, it's even causing me issues while trying to install eth_account.

here's what bitarray creator said - 

![image](https://user-images.githubusercontent.com/67156051/161749983-b24ccbff-c3dd-44d3-9e6f-be66bbe96e57.jpeg)

**Comment by kclowes** [comment]
Closed in #163
